packages/amplify-graphql-model-transformer/src/graphql-model-transformer.ts

```diff
@@ -1084,7 +1085,7 @@ export class ModelTransformer extends TransformerModelBase implements Transforme
 
   private isModelField = (name: string): boolean => (!!this.typesWithModelDirective.has(name));
 
-  private getNonModelInputObjectName = (name: string): string => `${name}Input`;
+  private getNonModelInputObjectName = (name: string): string => `${toUpper(name)}Input`;
```

Comment:
Looking at the issue, you may need to use 'PascalCase'. Could this break existing v2 apps?

Reply:
👋 @sundersc To the first half of the comment: actually `toUpper(word: string)` is a method that capitalizes the first letter in a given argument which is a String. 
whereas `toPascalCase(words: string[])` does the same but appends any given input either before the given argument and/or after, like: 
`const name = toPascalCase(['Model', type.name.value, 'Connection']);` `// output:- Model${#xyz}Connection`

So, in our use case we're just capitalizing the first letter of a given Embeddable type and appending the `Input` that is  generated in the `schema.graphql` of the build folder as a `NonModelInput` input type. I think this would not break any existing apps on V2 but provide CX opportunity to not worry about the capitalization of the embeddable type. 

Both "camelCased" Embeddable type & "camelCased" Model type names are currently respected by V1 transformer whereas only "camelCased" Model type names are currently supported in V2 transformer. So, this PR adds support to the former in V1 case i.e, support to "camelCased" Embeddable type names.

Reply:
In this case, we can either say `${toPascalCase(name, "Input")}` or `${toUpper(name)Input}` as both give the same output. I've tried it with below schema:
```
type multiWordstring @model {
  name: String!
  description: String!
  multiWordstringNonmodel: multiWordstringNonmodel
}

type multiWordstringNonmodel {
  name: String!
  description: String!
}
```
And the expected type name for non-model input is `MultiWordstringNonmodelInput` which is what this PR is generating. I would add a unit test for this if possible.

Reply:
I don't think it would break customers since the current experience is that we fail during push due to missing type.